config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+  - interval: 6h
+    name: check-dependency-stats-periodical
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    always_run: false
+    optional: true
+    spec:
```

Comment:
I'm not sure what [`decoration_config`](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/coverage.yaml#L22-L23) means. Do we need that here right before spec?

Reply:
if we used it in the presubmit then keep it, else remove it.